release.json

```diff
@@ -8,7 +8,7 @@
     "nightly": {
         "INTEGRATIONS_CORE_VERSION": "master",
         "OMNIBUS_SOFTWARE_VERSION": "5d4f6995c19b604d7fc876446e4350ce52b235fb",
-        "OMNIBUS_RUBY_VERSION": "f3fc847e03ba7081e266b2d333210ba129128a14",
+        "OMNIBUS_RUBY_VERSION": "33b72112967a61552f9841de6ad87b3024760d2d",
```

Comment:
Not really part of the review, but I'd tend to suggest you use your dev branch name here
It allows you to simply push to omnibus-ruby and restart the existing `datadog-agent` jobs for faster iterations, and will also make it clear that the corresponding work in omnibus isn't merged yet (and will allow the `do-not-merge` job to correctly kick in)